target_chains/ethereum/sdk/stylus/.github/workflows/check-wasm.yml

```diff
@@ -0,0 +1,36 @@
+name: check-wasm
```

Comment:
Looks like all these workflows already exist in `.github/workflows`, so they should be deleted from `target_chains/ethereum/sdk/stylus/.github/workflows`.